contributing/release-schedule.md

```diff
@@ -44,7 +44,7 @@ You can determine what changes are in a given build using one of the following:
 
 ### AMP Experimental and Beta Channels
 
-The _AMP Experimental Channel_ is a way to opt a browser into using the **experimental** release build of the AMP JS libraries. The Experimental Channel_ **may be less stable** and it may contain features not yet available to all users.
+The _AMP Experimental Channel_ is a way to opt a browser into using the **experimental** release build of the AMP JS libraries. The Experimental Channel\_ **may be less stable** and it may contain features not yet available to all users.
```

Comment:
The underscore after "The Experimental Channel" should be deleted.

Reply:
The preceding underscore was missing (it's supposed to be italicized), probably accidentally deleted during a round of editing. Re-added